src/main/resources/config.yml

```diff
@@ -92,6 +92,9 @@ motd:
     running: green                                       # Color designating a match which is currently running
     finished: red                                        # Color designating a match which has ended
 
+server-ping-data:
+  enabled: true
+
```

Comment:
TBH, we don't need a toggle for this. There's really no reason to disable it. Let's just always do it.